src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
```

Comment:
Don't fix container size (if there is no such a requirement). The width of 100% on the header may be unnecessary if the header is a block element, as it will naturally expand to the full width of its parent container.